features/lwipstack/LWIPInterfaceL3IP.cpp

```diff
@@ -0,0 +1,167 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "lwip/tcpip.h"
+#include "lwip/tcp.h"
+#include "lwip/ip.h"
+#include "netif/etharp.h"
+#include "lwip/ethip6.h"
+#include "netsocket/nsapi_types.h"
+#include "netsocket/L3IP.h"
+
+#include "LWIPStack.h"
+
+#if LWIP_L3IP
+
+err_t LWIP::Interface::l3ip_low_level_output(struct netif *netif, struct pbuf *p, const ip4_addr_t *ipaddr)
```

Comment:
This could be just l3ip_output().   Low level here might have meant something like ethernet/arp output which is sort of lower level compared to IP output.